app/src/main/java/com/example/weather/network/CachedCityDetails.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.weather.network
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "city_table")
```

Comment:
let's rename one of these so that both Cached data classes have different tableNames. we want them to have their own separate tables in the DB